src/bit_machine/mod.rs

```diff
@@ -588,6 +613,13 @@ impl<J: Jet> ExecTracker<J> for NoTracker {
     fn track_right(&mut self, _: Ihr) {}
 
     fn track_jet_call(&mut self, _: &J, _: &[UWORD], _: &[UWORD], _: bool) {}
+
+    fn track_dbg_call(&mut self, _: &Cmr, _: Value) {}
+
+    fn is_track_debug_enabled(&self) -> bool {
+        // Set flag to test frame decoding in unit tests
+        cfg!(test)
```

Comment:
Not ideal, but I don't have a better idea

Reply:
Hehe, I think it's ok. Agree that it looks pretty weird, but the nice thing is that `cfg(test)` is only ever set within this project, so we don't need to worry about this weirdness somehow being exposed to our dependencies.

Reply:
I guess the "right" way to do it is to add a whole nother `Tracker` type for use in tests. But I think this is fine for now.